kornia/augmentation/_2d/geometric/perspective.py

```diff
@@ -22,6 +22,10 @@ class RandomPerspective(GeometricAugmentationBase2D):
         align_corners: interpolation flag.
         keepdim: whether to keep the output shape the same as input (True) or broadcast it
                  to the batch form (False).
+        sampling_method: ``'basic'`` | ``'area_preserving'``. Default: ``'basic'``
+            If ``'basic'``, samples by translating the image corners randomly inwards.
+            If ``'area_preserving'``, samples by randomly translating the image corners in any direction.
+            Preserves area on average. See https://arxiv.org/abs/2104.03308 for further details.
```

Comment:
@Parskatt you can use bibtex. See: https://github.com/kornia/kornia/blob/master/docs/source/references.bib

it can be done in an upcoming PR